fbcv-base.obo

```diff
@@ -10538,16 +10537,154 @@ is_obsolete: true
 [Typedef]
 id: FBcv:0008000
 name: has_age
-namespace: FlyBase_miscellaneous_CV
 is_functional: true
 
+[Typedef]
+id: RO:0002314
+name: inheres in part of
+def: "q inheres in part of w if and only if there exists some p such that q inheres in p and p part of w." []
+property_value: IAO:0000116 "Because part_of is transitive, inheres in is a sub-relation of inheres in part of" xsd:string
+property_value: IAO:0000117 "Chris Mungall" xsd:string
+property_value: IAO:0000119 http://www.ncbi.nlm.nih.gov/pubmed/20064205
+property_value: RO:0001900 RO:0001901
+property_value: seeAlso "defining-property-chains-involving-reflexivity"
+holds_over_chain: inheres_in part_of
+transitive_over: part_of ! part_of
```

Comment:
Hmm, just spotted this in fbcv-base.obo. Does this mean that the name and definition are being removed when making -simple?

Reply:
Uuuuuu awesome! Big problem, but now how to fix. You can merge for now.

Reply:
This will be much more complicated than I thought.. I will make a ticket and we will ignore this for now.. Its not so problematic, but fixing it is.